extras/container/heketi-start.sh

```diff
@@ -21,6 +28,60 @@ fail() {
     exit 1
 }
 
+archive_db() {
+    # create an archive file for the previous copy of the db
+    # before the server starts up
+    mkdir -p "${HEKETI_DB_ARCHIVE_PATH}"
+
+    # only archive the db if the db content is valid
+    if ! "$HEKETI_BIN" db export --dbfile "${HEKETI_PATH}/heketi.db" --jsonfile - >/dev/null ; then
+        error "Unable to export db. DB contents may not be valid"
```

Comment:
Check: ensure db export returns a negative for all possible corrupt dbs.

Reply:
I tested it locally and if the db is corrupted (bolt can't read it) the command exited with 2. If the db doesn't exist it exits with 1 but we don't expect the function to be run in that case.